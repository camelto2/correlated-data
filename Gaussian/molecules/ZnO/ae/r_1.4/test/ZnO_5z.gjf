%mem=160GB
%cpu=0-31
%chk=5z.chk
%rwf=/scratch/04843/gwang18/zno/Gau-388513.rwf

#P Restart

