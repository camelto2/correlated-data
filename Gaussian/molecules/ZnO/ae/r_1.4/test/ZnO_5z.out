 Entering Gaussian System, Link 0=g16
 Initial command:
 /home1/apps/gaussian/16rA.03/g16/l1.exe "/tmp/Gau-75415.inp" -scrdir="/tmp/"
 Entering Link 1 = /home1/apps/gaussian/16rA.03/g16/l1.exe PID=     75416.
  
 Copyright (c) 1988,1990,1992,1993,1995,1998,2003,2009,2016,
            Gaussian, Inc.  All Rights Reserved.
  
 This is part of the Gaussian(R) 16 program.  It is based on
 the Gaussian(R) 09 system (copyright 2009, Gaussian, Inc.),
 the Gaussian(R) 03 system (copyright 2003, Gaussian, Inc.),
 the Gaussian(R) 98 system (copyright 1998, Gaussian, Inc.),
 the Gaussian(R) 94 system (copyright 1995, Gaussian, Inc.),
 the Gaussian 92(TM) system (copyright 1992, Gaussian, Inc.),
 the Gaussian 90(TM) system (copyright 1990, Gaussian, Inc.),
 the Gaussian 88(TM) system (copyright 1988, Gaussian, Inc.),
 the Gaussian 86(TM) system (copyright 1986, Carnegie Mellon
 University), and the Gaussian 82(TM) system (copyright 1983,
 Carnegie Mellon University). Gaussian is a federally registered
 trademark of Gaussian, Inc.
  
 This software contains proprietary and confidential information,
 including trade secrets, belonging to Gaussian, Inc.
  
 This software is provided under written license and may be
 used, copied, transmitted, or stored only in accord with that
 written license.
  
 The following legend is applicable only to US Government
 contracts under FAR:
  
                    RESTRICTED RIGHTS LEGEND
  
 Use, reproduction and disclosure by the US Government is
 subject to restrictions as set forth in subparagraphs (a)
 and (c) of the Commercial Computer Software - Restricted
 Rights clause in FAR 52.227-19.
  
 Gaussian, Inc.
 340 Quinnipiac St., Bldg. 40, Wallingford CT 06492
  
  
 ---------------------------------------------------------------
 Warning -- This program may not be used in any manner that
 competes with the business of Gaussian, Inc. or will provide
 assistance to any competitor of Gaussian, Inc.  The licensee
 of this program is prohibited from giving any competitor of
 Gaussian, Inc. access to this program.  By using this program,
 the user acknowledges that Gaussian, Inc. is engaged in the
 business of creating and licensing software in the field of
 computational chemistry and represents and warrants to the
 licensee that it is not a competitor of Gaussian, Inc. and that
 it will not use this program in any manner prohibited above.
 ---------------------------------------------------------------
  

 Cite this work as:
 Gaussian 16, Revision A.03,
 M. J. Frisch, G. W. Trucks, H. B. Schlegel, G. E. Scuseria, 
 M. A. Robb, J. R. Cheeseman, G. Scalmani, V. Barone, 
 G. A. Petersson, H. Nakatsuji, X. Li, M. Caricato, A. V. Marenich, 
 J. Bloino, B. G. Janesko, R. Gomperts, B. Mennucci, H. P. Hratchian, 
 J. V. Ortiz, A. F. Izmaylov, J. L. Sonnenberg, D. Williams-Young, 
 F. Ding, F. Lipparini, F. Egidi, J. Goings, B. Peng, A. Petrone, 
 T. Henderson, D. Ranasinghe, V. G. Zakrzewski, J. Gao, N. Rega, 
 G. Zheng, W. Liang, M. Hada, M. Ehara, K. Toyota, R. Fukuda, 
 J. Hasegawa, M. Ishida, T. Nakajima, Y. Honda, O. Kitao, H. Nakai, 
 T. Vreven, K. Throssell, J. A. Montgomery, Jr., J. E. Peralta, 
 F. Ogliaro, M. J. Bearpark, J. J. Heyd, E. N. Brothers, K. N. Kudin, 
 V. N. Staroverov, T. A. Keith, R. Kobayashi, J. Normand, 
 K. Raghavachari, A. P. Rendell, J. C. Burant, S. S. Iyengar, 
 J. Tomasi, M. Cossi, J. M. Millam, M. Klene, C. Adamo, R. Cammi, 
 J. W. Ochterski, R. L. Martin, K. Morokuma, O. Farkas, 
 J. B. Foresman, and D. J. Fox, Gaussian, Inc., Wallingford CT, 2016.
 
 ******************************************
 Gaussian 16:  ES64L-G16RevA.03 25-Dec-2016
                22-Mar-2018 
 ******************************************
 %mem=160GB
 %cpu=0-31
 SetSPE:  set environment variable "MP_BIND" = "yes"
 SetSPE:  set environment variable "MP_BLIST" = "0,1,2,3,4,5,6,7,8,9,10,11,12,13,14,15,16,17,18,19,20,21,22,23,24,25,26,27,28,29,30,31"
 Will use up to   32 processors via shared memory.
 %chk=5z.chk
 %rwf=/scratch/04843/gwang18/zno/Gau-388513.rwf
 ----------
 #P Restart
 ----------
 (Restarting at link  913)
 Leave Link  913 at Thu Mar 22 21:10:21 2018, MaxMem= 21474836480 cpu:               6.1 elap:               0.2
 (Enter /home1/apps/gaussian/16rA.03/g16/l913.exe)
 ILRest=           1.
 CIDS:  MDV= 21474836480.
 IFCWin=0 IBDFC=1 NFBD=     0     0 NFCmp=     0     0 NFFFC=     0     0
 Using original routines for 1st iteration.
 Using DD4UQ or CC4UQ for 2nd and later iterations.
 Keep R2 and R3 ints in memory in symmetry-blocked form, NReq=13558324717.
 FoFCou: FMM=F IPFlag=           8 FMFlag=           0 FMFlg1=           0
         NFxFlg=           0 DoJE=F BraDBF=F KetDBF=F FulRan=T
         wScrn=  0.000000 ICntrl=       600 IOpCl=  0 I1Cent=           0 NGrid=           0
         NMat0=    1 NMatS0= 147153 NMatT0=    0 NMatD0=    1 NMtDS0=    0 NMtDT0=    0
 Petite list used in FoFCou.
 CIDS: In Core Option IDoMem=           1.
 Iteration Nr.  19
 **********************
 DD1Dir will call FoFMem   1 times, MxPair=      1064
 NAB=   361 NAA=   171 NBB=   171.
 A-vector RMS=  6.0756800D-06 Max=  3.0050673D-03 conv= 1.00D-05.
 RLE energy=       -0.5912168767
 DE(Corr)=  -1.8292347     E(CORR)=     -1870.8218265     Delta=-2.46D-03
 NORM(A)=   0.10227909D+01
 Iteration Nr.  20
 **********************
 DD1Dir will call FoFMem   1 times, MxPair=      1064
 NAB=   361 NAA=   171 NBB=   171.
 A-vector RMS=  5.3218205D-04 Max=  3.0457929D-02 conv= 1.00D-05.
 RLE energy=       -0.5465152469
 DE(Corr)=  -1.9317473     E(CORR)=     -1870.9243391     Delta=-1.03D-01
 NORM(A)=   0.10104565D+01
 Iteration Nr.  21
 **********************
 DD1Dir will call FoFMem   1 times, MxPair=      1064
 NAB=   361 NAA=   171 NBB=   171.
 A-vector RMS=  9.1463128D-04 Max=  4.9181506D-02 conv= 1.00D-05.
 RLE energy=       -1.8287732698
 DE(Corr)=  -1.9280273     E(CORR)=     -1870.9206191     Delta= 3.72D-03
 NORM(A)=   0.11033606D+01
 Iteration Nr.  22
 **********************
 DD1Dir will call FoFMem   1 times, MxPair=      1064
 NAB=   361 NAA=   171 NBB=   171.
 A-vector RMS=  2.4713776D-04 Max=  9.2219215D-03 conv= 1.00D-05.
 RLE energy=       -1.8278628968
 DE(Corr)=  -1.8294320     E(CORR)=     -1870.8220238     Delta= 9.86D-02
 NORM(A)=   0.11007071D+01
 Iteration Nr.  23
 **********************
 DD1Dir will call FoFMem   1 times, MxPair=      1064
 NAB=   361 NAA=   171 NBB=   171.
 A-vector RMS=  6.5801266D-05 Max=  2.5166830D-03 conv= 1.00D-05.
 RLE energy=       -1.8281348293
 DE(Corr)=  -1.8279299     E(CORR)=     -1870.8205217     Delta= 1.50D-03
 NORM(A)=   0.11003578D+01
 Iteration Nr.  24
 **********************
 DD1Dir will call FoFMem   1 times, MxPair=      1064
 NAB=   361 NAA=   171 NBB=   171.
 A-vector RMS=  1.0079522D-04 Max=  4.3176286D-03 conv= 1.00D-05.
 RLE energy=       -1.8285118566
 DE(Corr)=  -1.8288022     E(CORR)=     -1870.8213940     Delta=-8.72D-04
 NORM(A)=   0.10992921D+01
 Iteration Nr.  25
 **********************
 DD1Dir will call FoFMem   1 times, MxPair=      1064
 NAB=   361 NAA=   171 NBB=   171.
 A-vector RMS=  3.3314572D-05 Max=  1.0461393D-03 conv= 1.00D-05.
 RLE energy=       -1.8281468285
 DE(Corr)=  -1.8280185     E(CORR)=     -1870.8206103     Delta= 7.84D-04
 NORM(A)=   0.10990904D+01
 Iteration Nr.  26
 **********************
 DD1Dir will call FoFMem   1 times, MxPair=      1064
 NAB=   361 NAA=   171 NBB=   171.
 A-vector RMS=  2.3373574D-05 Max=  1.1031550D-03 conv= 1.00D-05.
 RLE energy=       -1.8281852367
 DE(Corr)=  -1.8283141     E(CORR)=     -1870.8209059     Delta=-2.96D-04
 NORM(A)=   0.10988391D+01
 Iteration Nr.  27
 **********************
 DD1Dir will call FoFMem   1 times, MxPair=      1064
 NAB=   361 NAA=   171 NBB=   171.
